client/components/thumb.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var Thumb = (props) => {
+
+  return (
+    <div class="thumb" onclick={(e) => {this.bind(alert(this.props.styleDetails)) }}>
+      <img src={props.styleDetails.thumb} title={props.styleDetails.name}></img>
+      {/* <img onclick={(e) => {this.props.setStyleDetails(this.props.styleDetails) }} src={props.styleDetails.thumb} title={props.styleDetails.name}></img> */}
+    </div>
+  )
+}
+
```

Comment:
Looks good, only feedback I have for this file is the idea of destructuring your props to reduce the repetitive typing of props.